### DISCUSSION TITLE
Cisco SX550 and SG250

### DISCUSSION THREAD

**Discussion by tolasws** [napalm-automation/napalm#1844] [Q&A]
Hello All,

im trying to get napalm working in netbox for devices Cisco SMB SG250X and SX550X but i cant find appropriate driver, tested with ios and napalm_s350

only thing that for now works is by using  napalm_s350 driver on SX550X where i can only extract config but no lldp or status, same setup on SG250X does not work

i have also tried to do it using Telent by setting argument port to 23 but dose not work either.

am i missing something, are these devices supported?

**Comment by ktbyers** [comment]
@tolasws You should probably create an issue in the `napalm_s350` repository and see if the maintainer there is willing to help you. It is also possible that you might need to fix the issue yourself. The NAPALM IOS driver does not support the SG250X or the SX550X.
